module3/homework/vectorOfSharedPointers/vectorFunctions.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <memory>
+#include <vector>
+
+std::vector<std::shared_ptr<int>> generate(int);
+void print(std::vector<std::shared_ptr<int>>& newVector);
```

Comment:
As it is "print" function then it will not modify the content of the given vector, so maybe you can provide the parameter as a const reference?